src/components/diagrams/singleLineDiagram/single-line-diagram-content.js

```diff
@@ -291,7 +286,9 @@ function SingleLineDiagramContent(props) {
                     );
                     resetOneBusShortcircuitAnalysisLoader();
                 })
-                .finally(closeBusMenu());
+                .finally(
+                    dispatch(setComputationStarting(false)) && closeBusMenu()
+                );
```

Comment:
I just check, and [finally take a function/lambda as argument](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally)
```suggestion
                .finally(() => {
                    dispatch(setComputationStarting(false));
                    closeBusMenu();
                });
```

Reply:
fixed, moved `closeBusMenu` call elsewhere